test/e2e/tests/terms-of-use.spec.js

```diff
@@ -0,0 +1,45 @@
+const { strict: assert } = require('assert');
+const { convertToHexValue, withFixtures } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Terms of use', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('accepts the updated terms of use', async function () {
+    const firstOfJan = 1672574400;
```

Comment:
Nit: this date seems to be `Tue Jan 20 1970 09:36:14 GMT+0100 `

Reply:
I converted the date [here](https://www.epochconverter.com/timezones?q=1672574400) and the output was `Sunday, 1 January 2023 12:00:00`.

What conversion did you use to get that date?

Reply:
oh that's interesting! I used this `new Date(1672574400)` 

![Screenshot from 2023-04-28 14-23-02](https://user-images.githubusercontent.com/54408225/235146552-c7c50909-4f39-4855-897f-73ea407dcc4b.png)